beacon-chain/blockchain/service.go

```diff
@@ -79,10 +85,22 @@ func (c *ChainService) Start() {
 		return
 	}
 
+	beaconState, err := c.beaconDB.GetState()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Yikes! Can we have a more detailed error if we are crashing the process?

Reply:
ah , sorry for this I meant `log.Error(err)` here :expressionless: , will get it changed